roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -77,9 +76,14 @@
   - name: "Process support template"
     shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig process logging-support-template |  {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f -"
 
+  - name: "Process imagestream template"
+    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig process logging-imagestream-template -v {{ oc_process_values }} |  {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f -"
```

Comment:
According to Clayton `-v KEY1=VALUE1,KEY2=VALUE2` is a bug and will be removed in the future, hopefully not soon as it's currently documented in the --help output, but anyway my proposal should work long term.